sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md

```diff
@@ -24,7 +24,7 @@ Allows for the specified site to be prioritized.
 
 As part of [Advanced Tenant Rename](/sharepoint/change-your-sharepoint-domain-name#advanced-tenant-rename-preview), organizations can prioritize up to 4,000 sites for initial execution among all other sites in your organization as part of the overall rename operation.
 
-It is possible to start prioritizing sites only once the rename operation has been scheduled using the [Start-SPOTenantRename](/start-spotenantrename?view=sharepoint-ps) cmdlet.
+It is possible to start prioritizing sites only once the rename operation has been scheduled using the [Start-SPOTenantRename](Start-SPOTenantRename.md?view=sharepoint-ps) cmdlet.
```

Comment:
`start-spotenantrename` -- without the `.md` extension.

Use the [preview URL](https://review.learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spotenantrenamesiteprioritization?view=sharepoint-ps&branch=pr-en-us-595) to verify: current link (with the `.md`) will give you a 404